javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+function handleSubmit(event) {
+  event.preventDefault();
+  var elements = document.getElementById('contact-form').elements;
+  var formValues = {};
+  for (var i = 0; i < elements.length; i++) {
+    var item = elements.item(i);
+    formValues[item.name] = item.value;
+  }
+  console.log('formValues:', formValues);
+}
```

Comment:
Don't forget to reset the form:

---

![image](https://user-images.githubusercontent.com/16143771/178816565-2ac25bfa-eb85-413c-a4d9-cc97a618d7b3.png)